src/anemoi/training/data/dataset.py

```diff
@@ -93,6 +87,7 @@ def __init__(
         assert self.multi_step > 0, "Multistep value must be greater than zero."
         self.ensemble_dim: int = 2
         self.ensemble_size = self.data.shape[self.ensemble_dim]
+        self.grid_size = self.data.shape[-1]
```

Comment:
may be useful (easier to understand) if we defined a `self.grid_dim = -1` and used that instead? (like we do for the ensemble dim just above)

Reply:
agreed